src/app/views/query-response/queryResponse.styles.ts

```diff
@@ -18,7 +18,7 @@ export const queryResponseStyles = (theme: ITheme) => {
       alignItems: 'center',
     },
     link: {
-      color: theme.palette.blue,
+      color: theme.palette.themeDarkAlt,
```

Comment:
Noticed here that themeDarkAlt is applied, while the blueMid value was changed in the theme files. Does this still work?

Reply:
Wow, so I think I missed that I had changed the blueMid. Actually, I shall use the blueMid instead of the themeDarkAlt colour